test/common/helpers/api.js

```diff
@@ -13,7 +13,7 @@
  */
 'use strict';
 
-var test = require('../../test');
+var test = testSuite;
```

Comment:
This pattern loses the benefit of the more explicit name `testSuite` over `test`.